### PR TITLE
Fix ps 456

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
This will solve the welcome screen and the ipad visual issues.

### CODE REVIEW

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

File: lib/screens/tutorial_screen.dart [hunk 1]
```diff
@@ -61,57 +63,65 @@ class _TutorialScreenState extends State<TutorialScreen> {
...
+            padding: EdgeInsets.symmetric(horizontal: 0.22.sw),
+            child: Image.asset(
+              item[kImageTutorial],
+              height: isTablet ? 140.w : 200.w,
+              width: isTablet ? 140.w : 200.w,
+              fit: BoxFit.fill,
+            )),
+        SizedBox(height: 0.15.sh),
+        Text(item[kHeaderTutorial], style: TextStyle(fontSize: isTablet ? 16.sp : 18.sp, fontWeight: FontWeight.w800, color: EaselAppTheme.kDartGrey), textAlign: TextAlign.center),
+        const SizedBox(height: 15),
```
Comment (Amean) [inline lib/screens/tutorial_screen.dart]: Should SizedBox have .h?

File: lib/screens/tutorial_screen.dart [hunk 2]
```diff
@@ -61,57 +63,65 @@ class _TutorialScreenState extends State<TutorialScreen> {
...
+            padding: EdgeInsets.symmetric(horizontal: 0.22.sw),
+            child: Image.asset(
+              item[kImageTutorial],
+              height: 10.h,
+              width: 40,
+              fit: BoxFit.fill,
+            )),
+        SizedBox(height: 0.1.sh),
+        Text(item[kHeaderTutorial], style: TextStyle(fontSize: isTablet ? 16.sp : 18.sp, fontWeight: FontWeight.w800, color: EaselAppTheme.kDartGrey), textAlign: TextAlign.center),
+        const SizedBox(height: 15),
```
Comment (Amean) [inline lib/screens/tutorial_screen.dart]: Should SizedBox have .h?

File: lib/screens/tutorial_screen.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply lib/screens/tutorial_screen.dart]: Its .sh .It means 15 percent of the screen height.

File: lib/screens/tutorial_screen.dart (hunk 2 above)
Comment (kjawadDeveloper2) [reply lib/screens/tutorial_screen.dart]: its .sh. Means 10 percent of the screen height.

File: lib/screens/tutorial_screen.dart (hunk 1 above)
Comment (Amean) [reply lib/screens/tutorial_screen.dart]: I was looking at 113, sorry I should have been more clear

File: lib/screens/tutorial_screen.dart (hunk 2 above)
Comment (Amean) [reply lib/screens/tutorial_screen.dart]: Same here, I was looking at line 78, I should have put the line number

**Review by Amean — APPROVED** [review]

**Pull request merged**